make/istio/run

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+: "${GIT_ROOT:=$(git rev-parse --show-toplevel)}"
+source "${GIT_ROOT}/bin/common/versions.sh"
+source "${GIT_ROOT}/make/include/defaults"
+
+# Download and install the Istio release.
+istio_path="${GIT_ROOT}/.istio/${ISTIO_VERSION}"
```

Comment:
Well... if you destroy a Vagrant vm and bring a new one up, if you run `make istio-run` it will check for the version and prevent from downloading again the same version. If we "unversion", it's going to be dumb in the sense that whatever version is installed inside `//output/istio` is going to be picked.